app/javascript/components/Filters.js

```diff
@@ -5,6 +5,7 @@ import { Slider, Rail, Handles, Tracks, Ticks } from "react-compound-slider";
  * Component for a list of string-based filters, e.g. disease, species
  */
 function FilterList(props) {
+  if (typeof props.filters === 'undefined') return (<></>); // TODO: Remove this once /search/facets response is fixed
```

Comment:
What are you trying to do here?

Reply:
Good catch, this was a temporary stop-gap for an API issue that no longer exists.  I just removed it in c1e32ec.